crates/uv/tests/pip_compile_scenarios.rs

```diff
@@ -83,7 +83,7 @@ fn requires_incompatible_python_version_compatible_override() -> Result<()> {
                  exit_code: 0
                  ----- stdout -----
                  # This file was autogenerated by uv via the following command:
-                 #    uv pip compile requirements.in --index-url https://test.pypi.org/simple --find-links https://raw.githubusercontent.com/zanieb/packse/de0bab473eeaa4445db5a8febd732c655fad3d52/vendor/links.html --cache-dir [CACHE_DIR] --python-version=3.11
+                 #    uv pip compile requirements.in --find-links https://raw.githubusercontent.com/zanieb/packse/de0bab473eeaa4445db5a8febd732c655fad3d52/vendor/links.html --cache-dir [CACHE_DIR] --python-version=3.11
```

Comment:
Should `--find-links` be included?

Reply:
Yeah, good point!

Reply:
Ah, actually in `pip-tools` that's a separate option `--emit-find-links / --no-emit-find-links`, now I'm thinking maybe that makes sense 🤔

Reply:
Oh interesting haha uhh let's keep them separate for now to match them.